digiwf-cypress/cypress/e2e/TL-2-Anzeigen-Meine-Aufgaben.cy.js

```diff
@@ -0,0 +1,80 @@
+import vorgangStarten from "../pages/vorgangStarten"
+import meineAufgaben from "../pages/meineAufgaben"
+import exampleUserTask from "../pages/exampleUserTask"
+import userTask from "../pages/userTask"
+
+const numberOfTasks =21
+
+before(() => {
+    cy.login()
+})
+
+
+describe('Vorgaenge Anzeigen', () => {
+    it('passes', () => {
+        let pageSize = 20;
+
+        //Step 0
+        cy.log('Step 0');
+        meineAufgaben.openVorgangStarten(0,20);
+        vorgangStarten.findProcess("Example Usertask",0,20);
+        vorgangStarten.clickListElement("Usertask-Example");
+        exampleUserTask.setNumberOfTasks(numberOfTasks);
+        exampleUserTask.setUserName(Cypress.env('fullUsername'));
```

Comment:
ggf. Cypress.env als Funktion in eine andere Dateiauslagern

Reply:
done